bindings/ios/MEGASdk.h

```diff
@@ -8695,6 +8695,32 @@ typedef NS_ENUM(NSInteger, AffiliateType) {
 */
 - (void)sendEvent:(NSInteger)eventType message:(NSString *)message;
 
+#pragma mark - Banner
+
+/**
+ * @brief Requests a list of all Smart Banners available for current user.
+ *
+ * The response value is stored as a MegaBannerList.
+ *
+ * The associated request type with this request is MegaRequest::TYPE_GET_BANNERS
+ * Valid data in the MegaRequest object received in onRequestFinish when the error code
+ * is MegaError::API_OK:
+ * - MegaRequest::getMegaBannerList: the list of banners
+ *
+ * On the onRequestFinish error, the error code associated to the MegaError can be:
+ * - MegaError::API_EACCESS - If called with no user being logged in.
+ * - MegaError::API_EINTERNAL - If the internally used user attribute exists but can't be decoded.
+ * - MegaError::API_ENOENT if there are no banners to return to the user.
```

Comment:
Same here, better to convert the comments to the counterpart types in our binding project